### PR TITLE
Introduced temp tables in init sql script

### PR DESCRIPTION
**Pull request by aklochkova**
Fixes https://github.com/OHDSI/WebAPI/issues/727

### CODE REVIEW

File: src/main/resources/ddl/results/init_concept_hierarchy.sql [hunk 1]
```diff
@@ -71,19 +67,33 @@ LEFT JOIN (
...
+  condition_insert_temp.concept_name AS snomed_concept_name,
+	CAST('Condition' AS VARCHAR(20)) AS treemap,
+	condition_insert_temp.pt_concept_name,
+	condition_insert_temp.hlt_concept_name,
+	condition_insert_temp.hlgt_concept_name,
+	soc.concept_name AS soc_concept_name
+FROM #condition_insert_temp LEFT JOIN @vocab_schema.concept soc ON condition_insert_temp.concept_id = soc.concept_id;
+
+truncate table #condition_insert_temp;
+drop table #condition_insert_temp;
```
Comment (wivern) [inline src/main/resources/ddl/results/init_concept_hierarchy.sql]: SqlRender recognizes table drop using this rule `IF OBJECT_ID('@table', 'U') IS NOT NULL DROP TABLE @table`
and w/o 'if object_id` illegal code may be generated.

**Comment by wivern** [comment]
@aklochkova On which databases did you test init scripts?

**Comment by aklochkova** [comment]
@wivern on postgresql

**Comment by pavgra** [comment]
@aklochkova , could you provide performance comparison using classic relational DB (e.g. Postgres) and something like Impala / Redshift. I don't see performance improvement while observing the results of `EXPLAIN` in PG.

**Comment by anthonysena** [comment]
Closing this out since we're unclear how these changes improve performance. I'm going to close this out. Thanks!

**Pull request closed without merging**